oma_ddf_library_schema.json

```diff
@@ -108,24 +108,24 @@
                                 "abstract": { "type": "string" },
                                 "contentTags": { "$ref": "#/definitions/tags" }
                               },
-                              "required": [ "contentMediaType", "functionalArea", "title", "status", "version", "contentDate", "fileName", "url"
+                              "required": [ "contentID", "contentMediaType", "functionalArea", "title", "status", "version", "contentDate", "fileName", "url"
                               ],
                               "additionalProperties": false
                             }
                         }
                   },
                   "additionalProperties": false,
-                  "required": [ "version", "publicationDate", "status"
+                  "required": [ "versionID", "version", "publicationDate", "status"

```

Comment:
@jmudge I also added versionID, publisherID, contentID as require element. I forgot to add it before.